Sources/Helpers/OAAppSettings.m

```diff
@@ -2301,6 +2309,31 @@ - (OAProfileSetting *) getSettingById:(NSString *)stringId
     return [_registeredPreferences objectForKey:stringId];
 }
 
+- (void) resetAllProfileSettingsForMode:(OAApplicationMode *)appMode
+{
+    for (OAProfileSetting *value in [_registeredPreferences objectEnumerator].allObjects)
```

Comment:
Aren't all profile settings inherit from OAProfileSetting class? Also, what about other types of ProfileSetting?

Reply:
mark as resolved